src/main/java/appeng/container/slot/SlotCraftingTerm.java

```diff
@@ -199,7 +201,8 @@ private ItemStack craftItem( final EntityPlayer p, final ItemStack request, fina
 						{
 							super.onTake( p, is );
 							// actually necessary to cleanup this case...
-							p.openContainer.onCraftMatrixChanged( this.craftInv );
+							// TODO: is this ok? parameter seems to only be used by enchanting table
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)